### ISSUE TITLE
Add non-default file extensions to import statements

### ISSUE THREAD

**Issue by npbenjohnson** [testpossessed/auto-barrel#20]
Needed to support Vue (and I'm assuming JSX) barreling:
ErrorAlert.vue
generates:
``` js
export { default as ErrorAlert } from './errors/ErrorAlert';
```
needs to generate:
``` js
export { default as ErrorAlert } from './errors/ErrorAlert.vue';
```

**Comment by testpossessed** [comment]
When this feature was required I asked if the .vue extension was needed and was told by the requeter it was not.  I think I originally released it with the extension added but removed it when told otherwise. I have done some searching since you posted and everything I found suggests the extension is not needed because the Vue build tools understand what to look for.

I don't use Vue so relying very much on others to tell me how to support it.  If you can show me something definitive that shows it is required I will update the project and release a patched version.

**Comment by npbenjohnson** [comment]
It might be something worked around in webpack/ts/rollup config, or be difference between vue 3 and 4, but we do definitely require the extension. So, I understand if you don't want to add it if nobody else is asking for it, but if keeping the extension could at least be configurable that would be nice because adjusting the config across all of our vue projects and packages is non-trivial.

**Comment by testpossessed** [comment]
Published v1.9.0 that includes a new setting for providing a comma separated list of extensions that should be included when generating export statements.  It defaults to '.vue'.

Let me know if you have any issues.  Worked fine in local testing, but my experience with extensions is that doesn't always mean it will work in anger.
